From 405181dc10710b2c29931a0f44e72869f6ea3953 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Wed, 23 Mar 2016 14:52:41 +0000
Subject: [PATCH 111/122] init_task: faster timerslack

the default tuning is a compromise between client power and server performance;
for a server distro like Clear Linux, we don't need to compromise.
(for non-server usages we have different kernel binaries)

in principle this can be done as a patch to systemd as well, but we have a shared
systemd between usages while we have different kernels, so the logistics
for where the patch goes work out better here
---
 include/linux/init_task.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/init_task.h b/include/linux/init_task.h
index f8834f8..5fb7964 100644
--- a/include/linux/init_task.h
+++ b/include/linux/init_task.h
@@ -244,7 +244,7 @@
 	.journal_info	= NULL,						\
 	.cpu_timers	= INIT_CPU_TIMERS(tsk.cpu_timers),		\
 	.pi_lock	= __RAW_SPIN_LOCK_UNLOCKED(tsk.pi_lock),	\
-	.timer_slack_ns = 50000, /* 50 usec default slack */		\
+	.timer_slack_ns = 1000, /* 1 usec default slack */		\
 	INIT_TIMER_LIST							\
 	.pids = {							\
 		[PIDTYPE_PID]  = INIT_PID_LINK(PIDTYPE_PID),		\
-- 
2.9.2

